felix/daemon/daemon.go

```diff
@@ -369,6 +369,14 @@ configRetry:
 		simulateDataRace()
 	}
 
+	// We may need to temporarily disable encrypted traffic to this node in order to connect to Typha
+	if configParams.WireguardEnabled && configParams.WireguardHostEncryptionEnabled {
+		err := bootstrapWireguard(configParams.WireguardInterfaceName, configParams.FelixHostname, v3Client)
+		if err != nil {
+			log.WithError(err).Fatal("Couldn't bootstrap WireGuard host connectivity")
```

Comment:
Good idea to drop a `time.Sleep(2 * time.Second)` in here to avoid a tight restart loop